azuredeploy.json

```
@@ -0,0 +1,403 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "functionAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "Function App Nam. Must be globally unique."
+            },                        
+            "maxLength": 24
+        },
+        "storageAccountName": { 
+            "type": "string",
+            "metadata": {
+                "description": "Storage Account Name (lowercase). Must be globally unique"
+            },         
+            "minLength": 0,
+            "maxLength": 9
+        },
+        "hostingPlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "Hosting Service Name. Globally unique."
+            },            
+            "minLength": 3,
+            "maxLength": 24
+        },
+        "blobContainerPrefix": {
+            "type":"string",
+            "metadata": {
+                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region'"
+            },
+            "defaultValue":"region"
```

```suggestion
            "defaultValue":"region-"
```
Just want to be consistent with what the client app is using as the default.

---

The hyphen gets appended later on. This way they don't have to worry about it at all.